answerking_app/tests/test_unit/test_unit_productdetailserializer.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.utils.serializer_helpers import ReturnDict
+
+from answerking_app.models.serializers import ProductDetailSerializer
+from answerking_app.tests.test_unit.UnitTestBaseClass import UnitTestBase
+
+
+class ProductDetailSerializerTests(UnitTestBase):
+
+    utb = UnitTestBase()
+
+    serializer_path: str = "answerking_app.models.serializers."
+    test_product_serializer_data: dict = utb.get_fixture(
+        "products", "prod_id_only.json"
+    )
+    test_product_serializer: ProductDetailSerializer = ProductDetailSerializer(
+        test_product_serializer_data
+    )
+
+    def test_product_detail_serializer_contains_correct_fields(self):
+        data: ReturnDict = self.test_product_serializer.data
+        self.assertCountEqual(
```

Comment:
do assertEquals so we check names of fields also

Reply:
Done